contrib/format-maprdb/src/main/java/org/apache/drill/exec/store/mapr/db/json/JsonTableGroupScan.java

```diff
@@ -184,11 +184,9 @@ public MapRDBSubScan getSpecificScan(int minorFragmentId) {
 
   @Override
   public ScanStats getScanStats() {
-    //TODO: look at stats for this.
-    long rowCount = (long) ((scanSpec.getSerializedFilter() != null ? .5 : 1) * totalRowCount);
```

Comment:
Why usage of `JsonScanSpec` is no more relevant here?

Reply:
The totalrowcount already factored in the scanspec condition - please see the metatable call above. This was a bug in changes made for DRILL-5429. I missed it while reviewing the PR. The original rowcount did not factor the condition earlier hence it was estimating it here based on whether the filter is present or not.